src/components/connection-prompt/MicrobitWearingInstructionDialog.svelte

```diff
@@ -18,15 +19,14 @@
 </script>
 
 <main>
-  <div class="w-200">
-    <p class="font-bold text-2xl text-left leading-10">
+  <div class="w-175">
+    <DialogHeading>
       {#if currentStage === 'usb'}
         {$t('connectMB.wearingSetup.bluetooth.heading')}
       {:else if currentStage === 'usb1'}
         {$t('connectMB.wearingSetup.radioConnection.heading')}
-      {/if}
-    </p>
-    <div class="space-y-10 mt-5">
+      {/if}</DialogHeading>
```

Comment:
Just spotted a minor unexpected code formatting, not a merge blocker
```suggestion
      {/if}
    </DialogHeading>
```

Reply:
Thanks [3652721](https://github.com/microbit-foundation/ml-trainer/pull/114/commits/3652721d9b763a07bd419a0c3d68073f6021c679)